src/Sanderling/Sanderling.Exe/sample/script/Mine.ore.cs

```diff
@@ -243,13 +243,27 @@ Func<object>	DefenseStep()
 	
 	var RatTargetNext = SetRatTarget?.OrderBy(target => target?.DistanceMax ?? int.MaxValue)?.FirstOrDefault();
 	
+	//Populating array to check if drones are fighting or not
+	Sanderling.Interface.MemoryStruct.IUIElementText[] droneArray = Sanderling.MemoryMeasurementParsed?.Value?.WindowDroneView?.FirstOrDefault()?.LabelText.ToArray();
```

Comment:
@tetreum, looks like I confused the types in my earlier comment.
It should be `MemoryMeasurementParsed?.Value?.WindowDroneView?.FirstOrDefault()?.ListView`, as the `MemoryMeasurementParsed?.Value?.WindowDroneView` is a collection of windows.